mne/preprocessing/infomax_.py

```diff
@@ -148,7 +148,7 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
     # for extended Infomax
     if extended is True:
         signs = np.identity(n_features)
-        signs.flat[slice(0, n_features * n_subgauss, n_features)]
+        # signs.flat[slice(0, n_features * n_subgauss, n_features)] XXX ?
```

Comment:
looks like one assignment is missing or this remained from some earlier code. I bet the latter is the case, as tests pass as is.